internal/tokencache/access_token_cache_item_test.go

```diff
@@ -21,16 +23,16 @@ var (
 	extExpiresOn  = "1592049600"
 	cachedAt      = "1592049600"
 	atCacheEntity = &accessTokenCacheItem{
-		HomeAccountID:                  &testHID,
-		Environment:                    &env,
-		CredentialType:                 &credential,
-		ClientID:                       &clientID,
-		Realm:                          &realm,
-		Scopes:                         &scopes,
-		Secret:                         &secret,
-		ExpiresOnUnixTimestamp:         &expiresOn,
-		ExtendedExpiresOnUnixTimestamp: &extExpiresOn,
-		CachedAt:                       &cachedAt,
+		HomeAccountID:                  testHID,
```

Comment:
>HID [](start = 38, length = 3)

nit: same as earlier, i would write out homeAccountId, so it's easier to read quickly